src/main/resources/application-dev.yml

```diff
@@ -0,0 +1,24 @@
+server:
+  shutdown: immediate
+
+oauth:
+  endpoint.url: http://localhost:8090/auth
+
+management.endpoint:
+  health.cache.time-to-live: 0
+  info.cache.time-to-live: 0
+
+spring:
+  datasource:
+    url: 'jdbc:h2:mem:visit_scheduler_db;MODE=PostgreSQL'
+  h2:
```

Comment:
when do you use h2, is it running locally but the tests use Postgres? Should running locally yes postgres?

Reply:
will revisit this